flight/mcu/src/mcu_ac/main.cpp

```diff
@@ -22,78 +22,42 @@
  * Parth Shrotri
  * Jeffery Zhou
  * Karnap Patel
+ * Magilan Sendhil
```

Comment:
🙌🙌🙌

Reply:
I feel really proud of that macro, you have to realize.